packages/common/src/store/account/sagas.ts

```diff
@@ -107,23 +107,24 @@ export function* fetchAccountAsync({ isSignUp = false }): SagaIterator {
   yield* put(signedIn({ account, isSignUp }))
 }
 
-export function* cacheAccount(account: User) {
+export function* cacheAccount(account: AccountUserMetadata) {
+  const { user: accountUser } = account
   const localStorage = yield* getContext('localStorage')
 
   yield put(
     cacheActions.add(Kind.USERS, [
-      { id: account.user_id, uid: 'USER_ACCOUNT', metadata: account }
+      { id: accountUser.user_id, uid: 'USER_ACCOUNT', metadata: account }
     ])
   )
-
+  const { playlists: collections, ...user } = accountUser
   const formattedAccount = {
-    userId: account.user_id,
-    collections: [],
+    userId: accountUser.user_id,
+    collections,
     orderedPlaylists: []
```

Comment:
I don't think this is actually used? We can probably just remove it from the type expected by the local storage function and stop passing empty here all the time?